.github/workflows/test_dbt_fal.yml

```diff
@@ -0,0 +1,167 @@
+name: Test dbt-fal adapter
+
+on:
+  pull_request:
+    types: [assigned, opened, synchronize, reopened]
+    paths-ignore:
+      - "examples/**"
+      - "docsite/**"
+      - "README.md"
+      - "LICENSE"
+      - "docker/**"
+
+  push:
+    branches: [act1]
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        profile: ["postgres", "bigquery", "snowflake", "duckdb"]
+        python: ["3.8"]
+        environment: ["local", "venv", "conda"]
+        # bigquery implementation fails due to concurrency, let's
+        # limit it to 1 for now.
+        exclude:
+          - profile: "bigquery"
+            environment: "venv"
+          - profile: "bigquery"
+            environment: "conda"
+
+    # Run only the latest commit pushed to PR
+    concurrency:
+      group: "${{ github.head_ref || github.run_id }}-${{ github.workflow }}-${{ matrix.profile }}-${{ matrix.python }}-${{ matrix.environment }}"
+      cancel-in-progress: true
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: 'fal'
+
+      - uses: actions/checkout@v2
+        with:
+          repository: fal-ai/dbt-core
+          ref: act1
+          path: 'dbt-core'
+
+      - uses: actions/checkout@v2
+        with:
+          repository: isidentical/jaffle_shop_with_fal # change this to fal-ai/jaffle_shop_with_fal
+          ref: add-profiles-for-fal-ci  # change this to act1
+          path: 'jaffle_shop'
+
+      - name: Start Docker database
+        working-directory: fal/integration_tests
+        if: contains(fromJSON('["postgres"]'), matrix.profile)
+        run: docker-compose up -d
+
+      - name: Install conda
+        uses: s-weigand/setup-conda@v1
+        if: contains(fromJSON('["conda"]'), matrix.environment)
+        with:
+          activate-conda: false
+          python-version: ${{ matrix.python }}
+          conda-channels: anaconda, conda-forge
+
+      - name: Setup latest dependencies
+        run: |
+          python -m venv .venv
+          source .venv/bin/activate
+          pip install --upgrade pip
+          pip install build wheel
+
+          # First install the adapters. Since they depend on
+          # a fixed dbt-core version; if we install them after
+          # the dbt-core installation, they'll try to overwrite
+          # the existing revision.
+
+          pip install dbt-${{ matrix.profile }} -c fal/adapter/compatible_adapters.txt
+
+          pushd dbt-core
+          pip install -r dev-requirements.txt -r editable-requirements.txt
+          popd
+
+          pushd fal
+          pip install ".[${{ matrix.profile }}]"
```

Comment:
I think we don't need extras here, since we are taking care database read/write in the `fal/adapter` code.